pages/en/lb4/Testing-Your-Extensions.md

```diff
@@ -0,0 +1,303 @@
+---
+lang: en
+title: 'Testing your extension'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Testing-your-extension.html
+summary:
+---
+
+## Overview
+
+An extension for LoopBack 4 is likely to be used by thousands and as such it's
+important to have a thorough test suite for an extension because:
```

Comment:
is "to" better than "because" here?

Reply:
`because it`

All of the bullet points should be natural extensions of the leading sentence.

`because ensure your extension works correctly`
vs
`because it ensures your extension works correctly`

All that said, tests don't necessarily ensure that your extension works correctly, only that the existing user cases you've turned into tests are being satisfied.

Reply:
Switching to `to` instead of `because` cause the points become natural extensions that way. Rolls off the tongue smoother